language-service/src/services/yamlCompletion.ts

```diff
@@ -98,518 +98,523 @@ export class YAMLCompletion {
-				scanner.setPosition(child.end);
-				let token = scanner.scan();
-				if (token === Json.SyntaxKind.CommaToken && offset >= scanner.getTokenOffset() + scanner.getTokenLength()) {
-					return i + 1;
-				}
-				return i;
-			} else if (offset >= child.start) {
-				return i;
-			}
-		}
-		return 0;
-	}
-
-	private isInComment(document: TextDocument, start: number, offset: number) {
-		let scanner = Json.createScanner(document.getText(), false);
-		scanner.setPosition(start);
-		let token = scanner.scan();
-		while (token !== Json.SyntaxKind.EOF && (scanner.getTokenOffset() + scanner.getTokenLength() < offset)) {
-			token = scanner.scan();
-		}
-		return (token === Json.SyntaxKind.LineCommentTrivia || token === Json.SyntaxKind.BlockCommentTrivia) && scanner.getTokenOffset() <= offset;
-	}
-
-	private getInsertTextForPlainText(text: string): string {
-		return text.replace(/[\\\$\}]/g, '\\$&');   // escape $, \ and } 
-	}
-
-	private getInsertTextForValue(value: any, separatorAfter: string): string {
-		var text = value;
-		if (text === '{}') {
-			return '{\n\t$1\n}' + separatorAfter;
-		} else if (text === '[]') {
-			return '[\n\t$1\n]' + separatorAfter;
-		}
-		return this.getInsertTextForPlainText(text + separatorAfter);
-	}
-
-	private getInsertTextForProperty(key: string, propertySchema: JSONSchema, addValue: boolean, separatorAfter: string): string {
-
-		let propertyText = this.getInsertTextForValue(key, '');
-		// if (!addValue) {
-		// 	return propertyText;
-		// }
-		let resultText = propertyText + ':';
-
-		let value;
-		let nValueProposals = 0;
-		if (propertySchema) {
-			if (nValueProposals === 0) {
-				var type = Array.isArray(propertySchema.type) ? propertySchema.type[0] : propertySchema.type;
-				if (!type) {
-					if (propertySchema.properties) {
-						type = 'object';
-					} else if (propertySchema.items) {
-						type = 'array';
-					}
-				}
-				switch (type) {
-					case 'boolean':
-						value = ' $1';
-						break;
-					case 'string':
-						value = ' $1';
-						break;
-					case 'object':
-						value = '\n\t';
-						break;
-					case 'array':
-						value = '\n\t- ';
-						break;
-					case 'number':
-					case 'integer':
-						value = ' ${1:0}';
-						break;
-					case 'null':
-						value = ' ${1:null}';
-						break;
-					default:
-						return propertyText;
-				}
-			}
-		}
-		if (!value || nValueProposals > 1) {
-			value = '$1';
-		}
-		return resultText + value + separatorAfter;
-	}
-
-	private evaluateSeparatorAfter(document: TextDocument, offset: number) {
-		let scanner = Json.createScanner(document.getText(), true);
-		scanner.setPosition(offset);
-		let token = scanner.scan();
-		switch (token) {
-			case Json.SyntaxKind.CommaToken:
-			case Json.SyntaxKind.CloseBraceToken:
-			case Json.SyntaxKind.CloseBracketToken:
-			case Json.SyntaxKind.EOF:
-				return '';
-			default:
-				return '';
-		}
-	}
+            if (nodeParent && nodeParent.type === "object") {
```

Comment:
Has this been changed or it's all spacing?

Reply:
yes, just hide the spacing diff in the diff settings at the top.